traversal/scanner/CombinationFinder.java

```diff
@@ -72,7 +72,6 @@ public CombinationFinder(GraphManager graphMgr, Set<CombinationProcedure> proced
 
         for (CombinationProcedure procedure : procedures) {
             start(procedure);
-            if (procedure.vertices().size() == 1) continue;
```

Comment:
this is a bug that we discovered by simplifying the `TypeInference` class and was then revelead by a `TypeInferenceTest` which included `$a has $a`